evercrypt-rs/benches/benchmark.rs

```diff
@@ -0,0 +1,356 @@
+#[macro_use]
+extern crate criterion;
+extern crate evercrypt;
+extern crate rand;
+
+use criterion::Criterion;
+
+const DATA: &[u8; 1024] = &[1u8; 1024];
+
+pub fn randombytes(n: usize) -> Vec<u8> {
```

Comment:
I'll put this in the library later, using the Evercrypt randomness. But let's do it this way for now.